test/shared/axiosHttpClientTests.ts

```diff
@@ -3,9 +3,9 @@
 import * as assert from "assert";
 import * as should from "should";
 import { AxiosHttpClient } from "../../lib/axiosHttpClient";
-import { baseURL } from "../testUtils";
+import { isNode } from "../../lib/util/utils";
```

Comment:
sorting?

Reply:
According to the VS Code TypeScript extension's import sorting algorithm, this is the correct sorting order.